api/kubernetes/deploy_via_helm.sh

```diff
@@ -70,7 +70,7 @@ echo CHART_PACKAGE_VERSION: $CHART_PACKAGE_VERSION
 echo NAMEPSACE: $NAMESPACE
 echo API_VERSION: $API_VERSION
 
-export ARGO_VERSION="0.16.6"
+export ARGO_VERSION="0.22.15"
```

Comment:
As far as I know `deploy_via_helm.sh` is no longer used :thinking: 
@enocquet Can you confirm this please ?